tests/components/plugwise/test_init.py

```diff
@@ -43,3 +46,12 @@ async def test_smile_adam_xmlerror(hass, mock_smile_adam):
     mock_smile_adam.full_update_device.side_effect = Smile.XMLDataMissingError
     entry = await async_init_integration(hass, mock_smile_adam)
     assert entry.state == ENTRY_STATE_SETUP_RETRY
+
+
+async def test_unload_entry(hass, mock_smile_adam):
+    """Test being able to unload an entry."""
+    entry = await async_init_integration(hass, mock_smile_adam)
+
+    mock_smile_adam.async_reset = AsyncMock(return_value=True)
+    assert await async_unload_entry(hass, entry)
+    assert hass.data[DOMAIN] == {}
```

Comment:
```suggestion
    assert not hass.data[DOMAIN]
```

Reply:
btw, stole this from `hue/test_init.py` :)